python/cudf/cudf/_lib/copying.pyx

```diff
@@ -643,3 +644,16 @@ def shift(Column input, int offset, object fill_value=None):
         )
 
     return Column.from_unique_ptr(move(c_output))
+
+
+def get_element(Column input_column, int index):
```

Comment:
Not sure if you already tried this but would be good to just be consistent and then you could use `index` directly.
```suggestion
def get_element(Column input_column, size_type index):
```